tests/tests.js

```diff
@@ -287,6 +287,60 @@
 			'{"hello":"world","\\uD83D\\uDCA9":"foo","pile":"\\uD83D\\uDCA9"}',
 			'`toJSON` methods are not called when `json: false`'
 		);
+		equal(
+			jsesc('\u2192\xE9', {
+				'lowercaseHex': true
+			}),
+			'\\u2192\\xe9',
+			'Alphabetical hexadecimal digits are lowercase when `lowercaseHex: true`'
+		);
+		equal(
```

Comment:
I just noticed there’s only a single test with `lowercaseHex: true`. From this test on, please duplicate each of these tests with `lowercaseHex: true`.